### ISSUE TITLE
incomprehensible time format

### ISSUE THREAD

**Issue by AbelTaylor** [rancher/dashboard#5979]
**Rancher Server Setup**
- Rancher version: v2.6.5
- Installation option (Docker install/Helm Chart): docker

**Information about the Cluster**
- Kubernetes version: 1.23.6
- Cluster Type (Local/Downstream): local
  
**Describe the bug**
When I use view logs to view the pod logs, I see the following strange time format，there seems to be something that shouldn't be there
![image](https://user-images.githubusercontent.com/71317399/168937062-50fe6c85-7bc6-4c8f-b609-447e932a4708.png)
Below is my configuration
![image](https://user-images.githubusercontent.com/71317399/168937501-7d112a81-9c9b-4b1a-bd4c-a6dfac59b942.png)
Although I can choose the format of 2022-5-18 here, it will automatically become the format of map[label:...], I don't know if I have not set it correctly

**Comment by rak-phillip** [comment]
@AbelTaylor thanks for raising the issue 😁

@gaktive I think it's better to classify this one under Dashboard after taking a quick look

**Comment by catherineluse** [comment]
# QA Template
## Root cause
I added an `input` event to the LabeledInput component in this PR https://github.com/rancher/dashboard/pull/5850 without realizing that other components were listening for that event and expecting certain data to come with it. It caused the `input` event to be emitted too many times and without the right data being passed into it.
 
## What was fixed, or what changes have occurred
I changed the name of the event I added from 'input' to 'selecting' so that it no longer creates unwanted input events.
 
## Areas or cases that should be tested
I tested it by going through the same steps as to reproduce the issue. The steps are in this comment https://github.com/rancher/dashboard/pull/5989#issuecomment-1132417066

## What areas could experience regressions?
The change also affects the secret selector component used for receiver configs in AlertmanagerConfig custom resources. I tested that component locally to confirm that the secret selector still works. But I would say QA doesn't have to test that because the secret selector will be validated again soon because of this issue anyway: https://github.com/rancher/dashboard/issues/5875
 
## Are the repro steps accurate/minimal?

I believe so

**Comment by catherineluse** [comment]
The workaround for this issue is:

```
kubectl delete preferences.management.cattle.io -n [user-id] date-format
```

**Comment by AbelTaylor** [comment]
@catherineluse 
Can you cancel the time display in view logs? Application logs generally have time, so there is no need to display the time here.

**Comment by richard-cox** [comment]
> Can you cancel the time display in view logs? Application logs generally have time, so there is no need to display the time here.

@AbelTaylor That's a good request, can you create a separate github issue for it?

**Comment by AbelTaylor** [comment]
> > Can you cancel the time display in view logs? Application logs generally have time, so there is no need to display the time here.
> 
> @AbelTaylor That's a good request, can you create a separate github issue for it?

Of course @[richard-cox](https://github.com/richard-cox)
https://github.com/rancher/dashboard/issues/6015

**Comment by hameno** [comment]
@catherineluse For me the incorrect value was:
time-format              map[label:12:27:58 value:HH:mm:ss]

**Comment by sowmyav27** [comment]
**On 2.6.5 reproduced the issue:**
<img width="1308" alt="Screen Shot 2022-06-03 at 2 09 07 PM" src="https://user-images.githubusercontent.com/26032343/171953260-f3d8c66a-4192-4af7-9d2b-82cf36491eb8.png">

**On 2.6head commit id: `79a3bcb`**
This is seen fixed
<img width="1418" alt="Screen Shot 2022-06-03 at 2 30 00 PM" src="https://user-images.githubusercontent.com/26032343/171955875-32d9cef0-bf6e-4365-b804-066d836e8503.png">

**Comment by kajla** [comment]
Hello,
Still present in Rancher 2.6.6:
```
time-format              map[label:10:38:01 value:HH:mm:ss]
```

**Comment by kajla** [comment]
Ok, I have got the solution.
I manually edited the time-format value to this:
```
time-format      HH:mm:ss
```
It's a 24 hour format.

**Comment by kajla** [comment]
date-format also wrong in 2.6.6:
```
date-format              map[label:2022-08-29 value:YYYY-MM-DD]
```

**Comment by richard-cox** [comment]
@kajla This has been resolved in 2.6.7.

**Comment by kajla** [comment]
Thanks, I will update soon.
